APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
In this case, the method is tied to what I'm doing with the debugPrinter.  It's not intended to be a general-purpose method.  It's intended as a way of making sure I get consistent date formatting when I'm analyzing tasks and schedules, so we can compare dates across schedule printouts and know we're seeing dates in the same format.  It also lets me normalize the dates when I print a bunch of schedules in a columnar format.